src/main/java/io/vavr/jackson/datatype/deserialize/SeqDeserializer.java

```diff
@@ -46,6 +47,9 @@
         if (Array.class.isAssignableFrom(javaType.getRawClass())) {
             return Array.ofAll(result);
         }
+        if (IndexedSeq.class.isAssignableFrom(javaType.getRawClass())) {
```

Comment:
Hi @kennymacleod 

> Tests are failing, but seem to be failing on master also.

Just move this one to the end of the method (right before "default serialization"), and I believe that all tests will pass. The Vavr's `Vector` is `IndexedSeq` also.

Reply:
Good call. Done.